frontend/pendingTranslations.ftl

```diff
@@ -394,6 +394,21 @@ faq-question-bulk-trackerremoval-answer = You can only turn tracker removal on a
 faq-question-trackerremoval-breakage-question = Why do my emails look broken?
 faq-question-trackerremoval-breakage-answer-2 = Sometimes removing trackers may cause your email to look broken, because the trackers are often contained within images and links. When the tracker is removed, the email looks like it’s been formatted wrong because images are missing. This can’t be fixed for emails you’ve already received. If this is preventing you from reading your emails properly, turn off tracker removal.
 
+## VPN and Relay Bundle Banner
+#   $monthly_price (string) - the monthly cost (including currency symbol) for a given plan. Examples: $0.99, 0,99 €
+#   $old_price (string) - the outdated monthly cost (including currency symbol) for a given plan. This value has a strikethrough.
+#   #savings (string) - the percentage saved (including % symbol) for a given plan. Examples: 50%, 70%
```

Comment:
When submitting to the l10n repo, you'll want to place these above the string that they're used in, so that Pontoon can show them to the localisers when relevant.

Reply:
done